styles.css

```diff
@@ -0,0 +1,65 @@
+*{
+  box-sizing:border-box;
+}
+
+
+.u-list-reset {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+.u-list-inline li {
+  display: inline-block;
+  padding-left: 10px;
+  padding-left: 10px;
+}
+
+body {
+  background-color:hsl(234,18.5%,10.6%)
+}
+
+img {
+  float: left;
+}
+
+h2 {
+  font-family: "Cabin Sketch", cursive;
+  color: hsl(30, 4.3%, 91%);
+  font-size: 54px;
+}
+
+ul {
+  font-family: "Architects Daughter", cursive;
+  color: hsl(30, 4.3%, 91%);
+  font-size: 18px;
+  padding-bottom: 20px;
+}
+
+.container {
+  box-sizing: border-box;
```

Comment:
You already set `box-sizing` on line 2 of your CSS and do not need to repeat it here:
```suggestion
```